### ISSUE TITLE
Lag spikes when sprinting on modded block

### ISSUE THREAD

**Issue by Zekiu** [TheEpicBlock/PolyMc#101]
The modded block is from the mod [Blockus](https://www.curseforge.com/minecraft/mc-mods/blockus).
In F3 debug screen, it shows up as a noteblock. When you try and sprint on it, the FPS starts dropping on the client.
(Note: Sprinting on vanilla noteblocks doesn't cause this problem)

This is what shows up in the client log:
```
[15:59:05] [Render thread/WARN]: Could not spawn particle effect net.minecraft.class_2388@68024037
[15:59:05] [Render thread/WARN]: Could not spawn particle effect net.minecraft.class_2388@154d1c5d
[15:59:06] [Render thread/WARN]: Could not spawn particle effect net.minecraft.class_2388@60b0210
[15:59:06] [Render thread/WARN]: Could not spawn particle effect net.minecraft.class_2388@285de8ee
[15:59:06] [Render thread/WARN]: Could not spawn particle effect net.minecraft.class_2388@2dc1278b
[15:59:06] [Render thread/WARN]: Could not spawn particle effect net.minecraft.class_2388@4e46b1d1
[15:59:06] [Render thread/WARN]: Could not spawn particle effect net.minecraft.class_2388@698bed48
[15:59:06] [Render thread/WARN]: Could not spawn particle effect net.minecraft.class_2388@22ca645a
[15:59:06] [Render thread/WARN]: Could not spawn particle effect net.minecraft.class_2388@7977e761
[15:59:06] [Render thread/WARN]: Could not spawn particle effect net.minecraft.class_2388@6dc95be7
[15:59:06] [Render thread/WARN]: Could not spawn particle effect net.minecraft.class_2388@5a60ab94
```
and it repeats until the client stops sprinting on the block.
Walking normally on the block doesn't cause this problem.

I have the generated resource pack loaded on client.

Minecraft Version: 1.17.1
PolyMc Version: 3.2.0
FabricAPI Version: 0.37.2

**Comment by TheEpicBlock** [comment]
It appears like the blockstate for the particle isn't remapped correctly. Looking at the code it uses `Block.STATE_IDS.getRawId`. I probably missed adding a mixin there

**Comment by FrostBird347** [comment]
When sprinting on any modded blocks without a texture this happens.
Unlike Zekiu the game does not unfreeze for me when the player stops sprinting and instead stays frozen until the player is kicked from the server. 
Teleporting between dimensions and using /kill does not appear to unfreeze the client.

**Comment by Zekiu** [comment]
Your client logs probably show something like this:
```
[23:07:15] [Render thread/WARN]: COM exception: Invalid Query: SELECT PERCENTUSAGE FROM Win32_PerfRawData_PerfOS_PagingFile
[23:07:15] [Render thread/WARN]: Could not spawn particle effect hn@b3658f0
[23:07:18] [Render thread/WARN]: COM exception: Invalid Query: SELECT PERCENTUSAGE FROM Win32_PerfRawData_PerfOS_PagingFile
[23:07:18] [Render thread/WARN]: Could not spawn particle effect hn@1673bcc3
[23:07:20] [Render thread/WARN]: COM exception: Invalid Query: SELECT PERCENTUSAGE FROM Win32_PerfRawData_PerfOS_PagingFile
[23:07:20] [Render thread/WARN]: Could not spawn particle effect hn@45d504ee
```
It only updates after 3 seconds, so that's why it freezes the game, instead of causing lag spikes.
It seems like it's an issue with your (and also mine) Windows installation, not with PolyMc, as I got a different output on a fresh Windows virtual machine (the one shown in my first comment)

**Comment by FrostBird347** [comment]
When I am able to I will obtain the log. 
I am on Mac instead of Windows, so the error will not be exactly the same.

**Comment by TheEpicBlock** [comment]
The most likely cause for this has already been found; the packet isn't remapped correctly. The fps decrease likely comes from printing the warnings. I've got a fix, just want to do some stuff before pushing it. The freezing is probably a bug outside of PolyMc, but will hopefully be fixed as well.

**Comment by TheEpicBlock** [comment]
Should be fixed in 19e2faa52d288190a187e1c515a1d827b2a2e576
